Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV COMPOSER_PROCESS_TIMEOUT 3600
 
 RUN apt-get update -q \
   && apt-get install -y --no-install-recommends \
-  unzip git apt-transport-https wget ssh libxml2-dev gnupg2 unixodbc-dev libgss3
+  unzip git apt-transport-https wget ssh libxml2-dev gnupg2 unixodbc unixodbc-dev libgss3
```

Comment:
Microsoft ma nejaku novsiu/nekompatibilnu verziu, preto sa to musi nainstalovat explicitne skor.